### PR TITLE
Update fireblocks sdk js

### PR DESCRIPTION
**Pull request by vsiskin**
## Pull Request Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

- Add blockchainDescriptor as a sorting option to getOwnedNFTs and getNFTs methods

- Add search option to getOwnedNFTs

- Add collection ownership api

Fixes (link to the issue here)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Chore / Documentation
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Locally tested against Fireblocks API

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have added corresponding labels to the PR

### CODE REVIEW

**Comment by nirmalik** [comment]
test

**Review by lazars14-f — APPROVED** [review]

**Comment by vsiskin** [comment]
All tested

File: src/types.ts [hunk 1]
```diff
@@ -811,6 +820,12 @@ export interface TokenWithBalance extends Token {
     ownershipLastUpdateTime: number;
 }
 
+export interface CollectionOwnership extends NFTCollection {
+    standard?: string;
+    blockchainDescriptor: string;
+    contractAddress?: string;
```
Comment (idanya) [inline src/types.ts:826]: is contractAddress really optional? having only blockchain does not say much about ownership

**Review by idanya — CHANGES_REQUESTED** [review]

File: src/types.ts (hunk 1 above)
Comment (vsiskin) [reply src/types.ts:826]: We will have Solana in the future that doesn't have standard or address so we've putted it this way on nft-service in order not to be a breaking change

Comment (idanya) [reply src/types.ts:826]: I see. But no matter the blockchain, having just a descriptor does not say much, will there be other fields added?

Comment (vsiskin) [reply src/types.ts:826]: we have id, name, symbol and descriptor that are required for now, probably in the future there will be some other fields

**Review by idanya — APPROVED** [review]

**Pull request merged**